### DISCUSSION TITLE
Can we use Rasa instance, Rasa Model with Botfront ?

### DISCUSSION THREAD

**Discussion by toanparadox** [botfront/botfront#951] [Q&A]
Hi, can we use model build from Rasa, and integrate an instance from Rasa docker image with Botfront ?
Or we must import data and build a new model from Botfront, and use rasa-for-botfront docker image?
Can we use botfront nlu only ?

**Comment by pheel** [comment]
Botfront is only compatible with rasa-for-botfront at the moment. However by using the Export feature (and selecting a single language), it should be relatively easy to get your bot running with a regular Rasa instance.

**Reply by toanparadox** [reply]
Great, thank you
